lib/index.js

```diff
@@ -106,24 +110,25 @@ exports.plugin = {
             return reply.response(res.output.payload).code(statusCode);
           }
           // custom redirect https://github.com/dwyl/hapi-error/issues/5
-          if (config.statusCodes[statusCode]
-            && config.statusCodes[statusCode].redirect) {
+          var currentCodeConfig = config.statusCodes[statusCode];
```

Comment:
I _agree_ that this code can be tidied up. I wrote it a _very_ long time ago and it isn't great. 

However, changing quite a lot of code that is not relevant to the bugfix of the **`redirect url`** makes it more work to review the pull request. ⏳ 

This kind of refactoring absolutely has a place in a "refactoring" pull request, but not in a "bugfix" where the purpose is to change the _minimum_ amount of code in order to fix a _specific_ bug.

Reply:
ok, i understand you